bridge/client/app/_components/ktb-subscription-item/ktb-subscription-item.component.html

```diff
@@ -1,35 +1,13 @@
-<div fxLayout="column" *ngIf="subscription">
-  <div fxLayout="row">
-    <input fxFlex="200px" class="mr-2 item" placeholder="Subscription name" dtInput [(ngModel)]="subscription.name" required>
-    <span fxFlex="100px" class="bold mr-1">subscribes to:</span>
-    <dt-select [(ngModel)]="subscription.event" class="mr-2 item" placeholder="Choose your task" aria-label="Choose your task" required>
-      <dt-option *ngFor="let task of tasks" [value]="task" [textContent]="task"></dt-option>
-    </dt-select>
-    <dt-filter-field
-      [dataSource]="_dataSource"
-      [filters]="subscription.getFilter(_dataSource.data)"
-      (filterChanges)="filterChanged(subscription, $event)"
-      label="Filter by stage and service"
-      aria-label="Filter by stage and service"
-      clearAllLabel="Clear all"
-    ></dt-filter-field>
+<ng-container *ngIf="project && subscription as subscription">
+  <input class="item" placeholder="Subscription name" dtInput [(ngModel)]="name" required [readonly]="true">
```

Comment:
Does this need to be an input? Or will this be used in #4975 ?

Reply:
In this PR I already changed it to `span` instead of `input`. I'll also change it here